public/js/eventTopicLoadedService.js

```diff
@@ -20,7 +20,6 @@ define('arma3-slotting/eventTopicLoadedService', function () {
     };
 
     $(window).bind('action:topic.loaded', topicLoaded);
-    $(document).ready(topicLoaded);
```

Comment:
I need to find out why I had added this line sometime in 2019 :thinking:

Reply:
It can be left in as the new guard will deal with this triggering the topic twice but from my testing this wasn't needed